docs/user-manual/civiform-admin-guide/using-enumerator-questions-and-screens-in-a-program.md

```diff
@@ -89,3 +88,113 @@ Any question type can be a repeated question, including the enumerator type.
+
+4. Click **Create**.
+
+---
+
+##### Step 2: Create Repeated Questions
+Now, create the repeated questions for each piece of information you want to collect about each entity (in this case, `household member`). For this example, we'll create repeated questions for:
+
+**Name**:
+1. Create a new question of type **"Name"**.
+2. Fill in the details:
+   - **Question text**: `What is $this's full legal name?`.  (The **$this** token refers to the household member listed by the applicant.)
+
+   - **Administrative identifier**: `household_member_name`
+   - **Question enumerator**: `household_members` (This is the Administrative identifier of the Enumerator Question created in the previous step.) 
+
+3. Click **Create**.
+
+**Birth Date**:
+1. Create a new question of type **"Date"**.
+2. Fill in the details:
+   - **Question text**: `What is $this's date of birth?`
+   - **Administrative identifier**: `household_member_birthdate`
+   - **Question enumerator**: `household_members`
+3. Click **Create**.
+
+   Tip:  By asking for the date of birth and not age directly,  we can calculate the age, and the benefit is  that the value of age is only valid at a specific point of time and becomes stale as time goes on, whereas a person's date of birth stays valid.  a slightly shorter application for the applicants. 
+
+**Relationship**:
+1. Create a new question of type **"Dropdown"**.
+2. Fill in the details:
+   - **Question text**: `What is $this's relationship to the applicant?`
+   - **Administrative identifier**: `household_member_relationship`
+   - **Dropdown options**: "Head of Household", "Spouse", "Child", "Parent"
+   - **Question enumerator**: `household_members`
+3. Click **Create**.
+
+**Employment Status**:
+1. Create a new question of type **"Radio Button"**.
+2. Fill in the details:
+   - **Question text**: `What is $this's employment status?`
+   - **Administrative identifier**: `household_member_employment_status`
+   - **Radio button options**: "Employed", "Unemployed", "Retired", "Student"
+   - **Question enumerator**: `household_members`
+3. Click **Create**.
+
+---
+
+##### Step 3: Add Enumerator and Repeated Questions to Screens
+1. Go to the **Programs** tab and select the program you're working on.
+2. Click **Edit**.
+3. Create a new screen or select an existing screen with no other questions.
+   - A screen can only have an enumerator question if it is the only question on the screen.
+4. Add the enumerator question (`List all members of your household`) to this screen from the question bank.
+5. Click **Create repeated screen** to create a repeated screen associated with the enumerator screen.
+   - **Screen name**: `household_member_details`
```

Comment:
**suggestion:** This will soon be applicant facing, so let's make it: "Household member details"